src/glue/jobs/s3_to_json.py

```diff
@@ -552,20 +648,47 @@ def process_record(
                 "Manifest" not in f.filename and
                 f.file_size > 0
         ]
-        logger.debug("contents: %s", z.namelist())
-        logger.debug("non-empty contents: %s", non_empty_contents)
         for json_path in non_empty_contents:
+            with z.open(json_path, "r") as f:
+                line_count = sum(1 for _ in f)
+            basic_file_info = get_basic_file_info(file_path=json_path)
+            logger.debug(
+                    "Getting metadata",
+                    extra={
+                        **basic_file_info,
+                        "event.kind": "event",
+                        "event.category": ["process"],
+                        "event.type": ["start"]
```

Comment:
If we have a `start` event.type - should we have a a respective end after it's been finished?

https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html#ecs-event-type-end

Reply:
In general, perhaps. I actually think we should remove this log message altogether. This was one of the log messages that was originally just plaintext and I tried to convert it to ECS.  It made sense when I was just beginning to develop the script and seeing where things were breaking, but getting metadata is a very lightweight process. Adding 18 extra lines of logging code to say (begin parsing metadata from the file name) and (end parsing metadata from the filename) seems a bit extreme.